projects/storefrontlib/src/lib/checkout/components/multi-step-checkout/shipping-address/address-form/address-form.component.html

```diff
@@ -5,79 +5,101 @@
                 <div class="form-group">
                     <ng-container *ngIf="titles$ | async as titles">
                         <div *ngIf="titles.length !== 0">
-                            <label for="Title" aria-required="true" class="required">Title</label>
-                            <ng-select [searchable]="false" [clearable]="false" [items]="titles" bindLabel="name"
-                                bindValue="code" placeholder="Select Title..." (change)="titleSelected($event)">
-                            </ng-select>
+                            <label>
+                                <span class="label-content required" aria-required="true">Title</span>
+                                <ng-select [searchable]="false" [clearable]="false" [items]="titles" bindLabel="name"
+                                    bindValue="code" placeholder="Select Title..." (change)="titleSelected($event)">
+                                </ng-select>
+                            </label>
                         </div>
                     </ng-container>
                 </div>
                 <div class="form-group">
-                    <label for="FirstName" aria-required="true" class="required">First name</label>
-                    <input id="firstName" class="form-control" type="text" required placeholder="First Name"
-                        formControlName="firstName">
+                    <label>
+                        <span class="label-content required" aria-required="true">First name</span>
```

Comment:
`aria-required` not needed when we tag input as required
```suggestion
                        <span class="label-content required">First name</span>
```